homework/src/app.js

```diff
@@ -2,10 +2,192 @@
   /* Your homework goes here */
   
   /* STRINGS */
+  /*1*/ 
+  const myString = "hello,this,is,a,difficult,to,read,sentence";
+  
+  console.log(myString.length);
+
+  console.log(myString.replace(/,/g ,' '));
+
+
+/* 2*/
+
+
+  const str = 'dlroW olleH';
+  const reverse = str 
+  console.log(Array.from(str).reverse().join(''));
 
   /* ARRAYS */
 
+  const favoriteAnimals = ["blowfish", "capricorn", "giraffe"];
+
+  favoriteAnimals.push('turtle');
+
+  // ["blowfish", "capricorn", "giraffe", "turtle"]
+
+  console.log(favoriteAnimals);
+
+  favoriteAnimals.splice(1,0,'horse');
+
+  // ["blowfish", "horse", "capricorn", "giraffe", "turtle"]
+
+  console.log(favoriteAnimals);
+
+  favoriteAnimals.splice(3,1,);
+
+  // ["blowfish", "horse", "capricorn", "turtle"]
+
+  console.log(favoriteAnimals);
+
+
   /* OPERATORS */
 
+  /* 1 */
+  
+  0 == '0'  // javascript converts number to string so string + string = true
+  0 == []   // 0 is falsy, [] is also falsy so falsy + falsy = true
+  "0" == [] // string is truthy. [] is falsy so truthy + falsy = false
```

Comment:
this one is tricky. `[]` converts to empty string `""`, thus `"0" == ""` which is `false`

Reply:
""(empty string) is falsy so "" will be false. result will false because of it.